testapps/ServerlessWithYamlFunction/ServerlessWithYamlFunction.csproj

```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <DotNetCliToolReference Include="Amazon.Lambda.Tools" Version="2.1.4" />
+    <DotNetCliToolReference Include="Amazon.Lambda.Tools" Version="5.4.4" />
```

Comment:
You could actually delete this `ItemGroup`. It predates .NET global tools and is no longer used.

Reply:
Deleted

Reply:
If you do keep it, double check the version is correct.